src/vs/platform/environment/node/environmentService.ts

```diff
@@ -114,6 +114,9 @@ export class EnvironmentService implements IEnvironmentService {
 	@memoize
 	get sharedIPCHandle(): string { return `${getIPCHandlePrefix()}-${pkg.version}-shared${getIPCHandleSuffix()}`; }
 
+	@memoize
+	get nodeCachedDataDir(): string { return path.join(this.userDataPath, 'CachedData', pkg.version); }
```

Comment:
Yes - we could consider not enabling this for insiders at all. However, to get started and to collect feedback I'll keep it enabled.